3.0/alpine3.14/Dockerfile

```diff
@@ -1,7 +1,21 @@
-FROM alpine:3.12
+#
+# NOTE: THIS DOCKERFILE IS GENERATED VIA "apply-templates.sh"
+#
+# PLEASE DO NOT EDIT IT DIRECTLY.
+#
 
-RUN apk add --no-cache \
-		gmp-dev
+FROM alpine:3.14
+
+RUN set -eux; \
+	apk add --no-cache \
+		bzip2 \
+		ca-certificates \
+		gmp-dev \
+		libffi-dev \
+		procps \
+		yaml-dev \
+		zlib-dev \
+	;
```

Comment:
In the context of https://github.com/docker-library/ruby/issues/492, I'm trying to remember why we added all this here, and I cannot for the life of me remember why we added all these permanently. :sob: